client/extensions/woocommerce/style.scss

```diff
@@ -1,10 +1,57 @@
 .woocommerce {
 
+	.form-label {
+		margin-bottom: 16px;
+	}
+
 	.foldable-card.products__variation-card .foldable-card__content {
 		padding: 0;
 		border: 0;
 	}
 
+	.products__product-form-details {
+		.products__product-form-details-featured {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/13251/commits/9f6e9e2f91910bf0482a7032f163727fe16a7a79